Persistance/DataContext.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Domain;
+using System.Net.Http.Headers;
+using System.Text.Json;
+using Bogus;
+
+namespace Persistance;
+
+public class DataContext : DbContext
+{
+    public DataContext()
+    {
+    }
+
+    public DataContext(DbContextOptions options) : base(options)
+    {
+    }
+
+    public DbSet<Movie> Movies { get; set; }
```

Comment:
We should bring DbSets to the top, when I go to the DbContext class I want to see with what I'm working for